karapuzio/assignment2/q1_ancestors/BinaryTree.java

```diff
@@ -0,0 +1,135 @@
+package edu.codeU.assignment2.Ancestors;
+
+import edu.codeU.assignment2.BinarySearchElementExeption;
+import edu.codeU.assignment2.BinaryTreeInitializationExceprion;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+
+public class BinaryTree{
```

Comment:
Generally interesting approach but can you please solve this without String parsing for direction? This seems like an unnecessary complication for a tree algorithm.